bundles/TinymceBundle/README.md

```diff
@@ -6,6 +6,12 @@
 TinyMCE bundle provides TineMCE as WYSIWYG-editor.
 Similar to Textarea and Input you can use the WYSIWYG editable in the templates to provide rich-text editing.
 
+## Installation
+
+```bash
+bin/console pimcore:bundle:install PimcoreTinymceBundle
+```
```

Comment:
wait, this bundle actually don't need to be installed, as there is nothing to install

https://github.com/pimcore/pimcore/blob/fbe03e30713eb2a9810c252837389ab96f32452d/bundles/TinymceBundle/src/Installer.php

see
![image](https://github.com/pimcore/pimcore/assets/6014195/1be733eb-d155-4e7e-9278-1c938124d7be)

The way to activate it is to add this to `config/bundles.php`
https://github.com/pimcore/demo/blob/4cd83c28782af1116a19bf2546c2d1c0b7489ad6/config/bundles.php#L42